Gruntfile.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-env node */
+"use strict";
+
+module.exports = function (grunt) {
+    grunt.initConfig({
+        lintAll: {
+            sources: {
+                md:    [ "./*.md","./docs/**/*.md"],
```

Comment:
The `docs` one was bad copypasta from the project I copied the file from.  Please erase my shame.

Reply:
Done, thanks for catching this.